srcpkgs/asusctl/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'asusctl'
+pkgname=asusctl
+version=6.0.11
+revision=1
+archs="x86_64* i686*" # fails to build on other archs, but this is utility for ROG laptops which are only x86_64
+build_style="gnu-makefile"
```

Comment:
use the cargo build style instead, probably

(you can do `make install-data` in post_install)